tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -211,60 +213,62 @@ that covers 800 nm-805 nm might have a FWHM of 2.5 and a wavelength value of 802
 
 The HDF5 dataset that we are working with in this activity contains more 
 information than we need to work with. For example, we don't necessarily need 
-to process all 426 bands - if we are interested in creating a product like NDVI
-which only users bands in the near infra-red and red portions of the spectrum. 
-Or we might only be interested in a spatial subset of the data - perhaps a region 
-where we have plots in the field.
+to process all 426 bands available in a full NEON hyperspectral reflectance file 
+- if we are interested in creating a product like NDVI which only users bands in 
+the near infra-red and red portions of the spectrum. Or we might only be interested 
+in a spatial subset of the data - perhaps a region where we have plots in the field.
 
 The HDF5 format allows us to slice (or subset) the data - quickly extracting the 
 subset that we need to process. Let's extract one of the green bands in our 
-dataset - band 34. By the way - what is the center wavelength value associated 
-with band 34? hint `wavelengths[34]`. How do we know this band is a green band 
+dataset - band 9. By the way - what is the center wavelength value associated 
+with band 9? hint: `wavelengths[9]`. How do we know this band is a green band 
 in the visible portion of the spectrum?
 
 ```{r get-reflectance-shape}
+# First, we need to extract the reflectance metadata:
+reflInfo <- h5readAttributes(f, "/SJER/Reflectance/Reflectance_Data")
+reflInfo
 
-# note that we can grab the dimensions of the dataset from the attributes
-# we can then use that information to slice out our band data
-nRows <- reflInfo$row_col_band[1]
-nCols <- reflInfo$row_col_band[2]
-nBands <- reflInfo$row_col_band[3]
+# Next, we parse out the different attributes using strsplit
```

Comment:
I don't see strsplit being used.

Reply:
You're right! That has changed for this updated dataset. I edited this to correct.